package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "single-spa",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
I prefer doing this with `yarn publish`, which also creates an annotated git tag automatically. And then pushing straight to master after publishing.

Now that you have perms, want to publish it yourself?